frontend/dispatcher.lua

```diff
@@ -196,11 +196,11 @@ local settingsList = {
     ----
 
     -- Reflowable documents
-    set_font = {category="string", event="SetFont", title=_("Set font"), rolling=true, args_func=require("fontlist").getFontArgFunc,},
+    set_font = {category="string", event="SetFont", title=_("Font Face"), rolling=true, args_func=require("fontlist").getFontArgFunc,},
```

Comment:
Lowercase words in title is how we do it elsewhere :-)
```suggestion
    set_font = {category="string", event="SetFont", title=_("Font face"), rolling=true, args_func=require("fontlist").getFontArgFunc,},
```